projects/1.20/assets/blue-skies/blue_skies/lang/zh_cn.json

```
@@ -0,0 +1,1354 @@
+	"advancements.blue_skies.everbright.get_wood.desc": "获得一块永昼之地的原木。",
+	"advancements.blue_skies.blue_sky_pie": "我的~，我的~，蔚蓝天空下的派~！",
+	"advancements.blue_skies.blue_sky_pie.desc": "制作一份彗星莓派。",
+	"advancements.blue_skies.regrowth": "可再生资源",
+	"advancements.blue_skies.regrowth.desc": "获得彗星树苗。",
+	"advancements.blue_skies.everbright.regen_brewberry": "粉色娇嫩",
+	"advancements.blue_skies.everbright.regen_brewberry.desc": "种植粉红酿莓并尝一口。",
+	"advancements.blue_skies.keys": "拿到一个，还差三个",
+	"advancements.blue_skies.keys.desc": "从惑乱地牢中获得一把钥匙。",
+	"advancements.blue_skies.everbright.kill_summoner": "致命魔法！",
+	"advancements.blue_skies.everbright.kill_summoner.desc": "战胜召唤师。",
+	"advancements.blue_skies.everbright.kill_starlit_crusher": "挑战自然！",
+	"advancements.blue_skies.everbright.kill_starlit_crusher.desc": "战胜星荧粉碎者。",
+	"advancements.blue_skies.everdawn.kill_alchemist": "居然还挺强！",
+	"advancements.blue_skies.everdawn.kill_alchemist.desc": "战胜炼金师。",
+	"advancements.blue_skies.everdawn.kill_arachnarch": "降维打击！",
+	"advancements.blue_skies.everdawn.kill_arachnarch.desc": "战胜幽咽蛛王。",
+	"advancements.blue_skies.enter_blinding_dungeon": "巫师之塔",
+	"advancements.blue_skies.enter_blinding_dungeon.desc": "进入惑乱地牢。",
+	"advancements.blue_skies.everbright.enter_nature_dungeon": "自然中枢",
+	"advancements.blue_skies.everbright.enter_nature_dungeon.desc": "进入自然地牢。",
+	"advancements.blue_skies.everdawn.enter_poison_dungeon": "蜘蛛洞窟",
+	"advancements.blue_skies.everdawn.enter_poison_dungeon.desc": "进入猛毒地牢。",
+	"advancements.blue_skies.get_moonstone_shard": "点亮世界！",
+	"advancements.blue_skies.get_moonstone_shard.desc": "挖掘月光石矿石以获得月光石碎片。",
+	"advancements.blue_skies.everbright.summon_golem": "人工伙伴",
+	"advancements.blue_skies.everbright.summon_golem.desc": "在召唤台中使得召唤之书能召唤人工傀儡。",
+	"advancements.blue_skies.everdawn.use_alchemy_table": "炼金术的力量！",
+	"advancements.blue_skies.everdawn.use_alchemy_table.desc": "使用炼金台。",
+	"advancements.blue_skies.everdawn.alchemy_gold": "终于。",
+	"advancements.blue_skies.everdawn.alchemy_gold.desc": "用炼金术点金。",
+	"advancements.blue_skies.everbright.use_tool_box": "处处修！",
+	"advancements.blue_skies.everbright.use_tool_box.desc": "使用工具盒。",
+	"advancements.blue_skies.everbright.use_falsite_in_tool_box": "额外的耐久度！",
+	"advancements.blue_skies.everbright.use_falsite_in_tool_box.desc": "用工具盒将谬铁和工具结合。",
+	"advancements.blue_skies.craft_food_prep_table": "炉火纯青",
+	"advancements.blue_skies.craft_food_prep_table.desc": "合成小灶。",
+	"advancements.blue_skies.everbright.craft_snowcap_oven": "现在终于开始用……蘑菇做菜了？",
+	"advancements.blue_skies.everbright.craft_snowcap_oven.desc": "合成雪帽熔炉。",
+	"advancements.blue_skies.everdawn.craft_horizonite_forge": "预热一下",
+	"advancements.blue_skies.everdawn.craft_horizonite_forge.desc": "合成炙铁熔炉。",
+	"advancements.blue_skies.everbright.kill_starlit_crusher_special": "扯下树皮",
+	"advancements.blue_skies.everbright.kill_starlit_crusher_special.desc": "用拳头给予星荧粉碎者最后一击。",
+	"advancements.blue_skies.everdawn.kill_arachnarch_special": "危机四伏",

```

```suggestion
	"advancements.blue_skies.everdawn.kill_arachnarch_special": "无处藏身",
```
https://movie.douban.com/subject/25724979//